frontend/package.json

```diff
@@ -71,23 +75,81 @@
   "devDependencies": {
     "env-cmd": "^10.1.0",
     "eslint-config-prettier": "^8.8.0",
+    "eslint-import-resolver-alias": "^1.1.2",
```

Comment:
"eslint-plugin-import": "^2.27.5", is lost from 29dc1c899cdd71ebd52005cb7111b4ad85d8f344 (#411)